### ISSUE TITLE
Some questions about code

### ISSUE THREAD

**Issue by orihnec** [Reza-Asiyabi/Complex-Valued-End-to-End-Deep-Network-for-SAR#4]
1. In the RV section, it seems that your code defines a variable self.image_max = image_max in the loss function that is not used at all. Could you explain the purpose of this variable?
2. In the loss function of the CV section, the loss will calculate a complex loss based on your structure, but complex loss cannot be computed in PyTorch. Could you explain how to solve this issue?
3. How was your data sampled? This does not seem to be reflected in the code.
4. After sampling the data according to your method, I found that the accuracy of the training set indeed met the requirements, but the accuracy of the test set is only around 76%. The model's generalization ability seems a bit poor. Could it be due to something I have not configured properly?
